src/plot_api/plot_config.js

```diff
@@ -48,8 +48,11 @@ module.exports = {
     // new users see some hints about interactivity
     showTips: true,
 
-    // enable/disable direct range entry at the pan/zoom drag points
-    enableRangeEntry: true,
+    // enable axis pan/zoom drag handles
+    showAxisDragHandles: true,
```

Comment:
A straight-up boolean is fine for now.

I'm thinking down the road, we could allow users to specify which drag handles to show for example:

```
showAxisDragHandles: 'n+s'
```

would only show the north (or top) and south (or bottom) dragger.

But that's for another day. :+1: